src/calendar-list/index.js

```diff
@@ -62,6 +66,7 @@ class CalendarList extends Component {
     scrollEnabled: true,
     scrollsToTop: false,
     removeClippedSubviews: Platform.OS === 'android' ? false : true,
+    keyboardShouldPersistTaps: 'never',
```

Comment:
This is the FlatList default value so no need to set it (if the prop will be passed as undefined this will be its value).